docs/GraFx-Studio/concepts/output-tasks/index.md

```diff
@@ -0,0 +1,95 @@
+# Output Tasks
+
+## Overview
+
+**Output Tasks** represent each job initiated to create output within GraFx Studio. This concept applies to both output jobs started from Studio UI, Template Designer Workspace and those triggered through the API.
+
+```sh
+/api/v1/environment/{environment}/output/pdf
+```
+
+Each output job is listed as an **Output Task** in the system, providing visibility into the status and details of all output jobs. The **Output Task List** allows users to monitor and manage the output tasks.
+
+## Accessing the Output Task List
+
+The **Output Task List** can be accessed through the **Environment Settings**. Here, users (Subscription Admins & Environment Admins) can view all the output jobs associated with the current environment.
+
+![screenshot-full](ot1.png)
+
+![screenshot-full](ot2.png)
+
+## Output Task List Columns
+
+The Output Task List contains several key details for each job, organized in the following columns:
+
+- **Name**: Displays the name of the document associated with the output job.
+- **Status**: Shows the current status of the output task. Possible statuses include:
+  - *Started*
+  - *Pending*
+  - *Finished*
+- **Format**: Indicates the output format, such as PDF, PNG, MP4, etc.
+- **Records**: Reflects the number of records in the current batch. For example, if a batch of 100 records is split into 4 tasks of 25 each, the user will see four separate output tasks listed.
+- **Date Created**: The date and time when the output job was initiated.
+
+![screenshot-full](ot2.png)
+
+Behind the "..." menu, you can also 
+
+    - Copy Task ID
+    - Copy User Information
+    - COpy Row Information
+    
+![screenshot](ot3.png)
+    
+**Task ID** Copies the unique ID (GUID) used for the task. Using API calls, this task ID is used to poll the status of the task.
+
+```JSON
+    84ad7436-f82f-4e87-8cd3-37299e37878c
+```
+
+**User Information** provides a JSON structure with the information about the user started the task.
+
+```JSON
+{
+    "id":"samlp|chili-publish-prd|lfB36OFPlzFI-wKmwAb0cZx4hlL--1234",
+    "email":"someuser@chili-publish.com",
+    "firstName":"Some",
+    "lastName":"User",
+    "avatar":"https://example.com/someuser.png",
+    "lastSeen":1726227211,
+    "status":"Pending",
+    "subscriptionRoles":["SA"],
+    "environmentRoles":null,"hasFullAccess":true,
+    "createdAt":1689165295,
+    "isFederated":true,
+    "membershipType":"Invitation"
+}
+```
+
+**Row Information**
+```JSON
+{
+    "id":"4095149a-745e-465b-95be-5b0518321a14",
+    "name":"Businesscard",
+    "dateCreated":"2024-09-18T15:35:17.759154",
+    "dateProcessed":"2024-09-18T15:35:19.079",
+    "format":"PNG",
+    "totalRecords":1,
+    "status":"Finished",
+    "source":"Template",
+    "owner":"samlp|chili-publish-prd|lfB36OFPlzFI-wKmwAb0cZx4hlL--qo3J7NqTmSZAu0",
+    "outputAvailable":false
+}
+```
+
+## Understanding Output Tasks
```

Comment:
Extra to add:
- Output generated directly from Template - User can access outputs of other Users
- Output generated from Project - User cannot access outputs of other Users